uncertainties/formatting.py

```diff
@@ -0,0 +1,1087 @@
+from math import isinf, isnan
+import math
+import re
+import warnings
+
+
+def isinfinite(x):
+    # TODO: Usages of this function should be replaced with not math.isfinite.
```

Comment:
This came from a code block that was providing backwards compatibility for Python before 3.2. It could just be `from math import isinfinite` now.